app/src/main/java/com/iic/mokojin/activities/PlayerQueueFragment.java

```diff
@@ -101,15 +101,17 @@ public void run() {
                     scheduleUpdateClock();
                     return;
                 }
-                getActivity().runOnUiThread(new Runnable() {
-                    @Override
-                    public void run() {
-                        if (null != mQueueAdapter) {
-                            mQueueAdapter.notifyDataSetChanged();
-                            scheduleUpdateClock();
+                if (null != getActivity()) {
```

Comment:
Hmm... are we sure that this code is being run on the correct lifecycle event? Maybe we should run `scheduleUpdateClock` on `onActivityCreated` or something?